### ISSUE TITLE
audio asset's start time in stream is beginning of file regardless of selected clip's start time

### ISSUE THREAD

**Issue by hburgund** [roundware/roundware-server#197]
It appears that when an audio asset is selected to play in a stream, a clip from that asset is properly selected per the `audiotrack` settings, but instead of playing just that clip once the dead air time is up, it plays the entire file from the beginning, silently, until the selected clip time arrives and then fades in at that point.

The result of this is vastly increased dead air (and perceived latency) and some strange behaviors when particularly long files are used.  I noticed the issue because of a project that had a number of assets that were several minutes in length, including:

```
http://stratfordhall.dyndns.org/rwmedia/20140825-133216.wav
```

which I imported into my vagrant VM to verify and test and I am getting the same behavior.  It is easiest to reproduce if you set the min/max duration times on the audiotrack to fairly small values, say 10-15s, in relation to the length of the long assets.  Then you will see the asset selected for playback in the logs well before you hear any part of the asset audibly.  And you will notice that if it takes a long time to hear the asset play, you will hear a clip from the asset that is further along in the file.

I'm not sure exactly what is happening, but there is no doubt that something is delaying the audible playback of the assets and it seems to be related to the length of the asset and the specific clip selected for playback.  Perhaps this is related to the change between `wavparse` `decodebin` `decodebin2`?  Seems like perhaps this is another unintended consequence of that, but that's just a guess.

**Comment by eosrei** [comment]
Could be related to the change from `gnomevfs` to `decodebin2`. Testing now.

**Comment by eosrei** [comment]
Gstreamer Seeking details: http://gstreamer.freedesktop.org/data/doc/gstreamer/head/manual/html/section-eventsseek.html

**Comment by jslootbeek** [comment]
I just confirmed that this was introduced by the change from wavparse to decodebin2 by PR #175. decodebin2 takes a considerable longer time to seek the requested segment of the wav file (as mentioned in the documentation: .
reverting `self.decodebin = gst.element_factory_make("decodebin2")` to `self.decodebin = gst.element_factory_make("wavparse")` removes the delay, but this would likely regress #170. At this time I don't know enough to determine whether wavparse and decodebin2 can be used together, but I will continue investigating.
It appears somebody else also found this an issue: http://gstreamer-devel.966125.n4.nabble.com/Prerolling-and-The-Segment-event-getting-delayed-after-the-Seek-event-td4676307.html I've posted a reply to see if this issue was solved, and to provide some additional context.
